gravitee-apim-distribution/pom.xml

```diff
@@ -38,16 +38,15 @@
         To add a new plugin in the bundle:
          1. add the version in the <properties> section.
             please respect alphabetic order and the separation between Gateway only, Management only and both
-         2. update the maven-dependency-plugin so the new plugin can be copied in the right temp folder (used in the packaging).
-            It can be apim-plugins, connectors, fetchers, policies, ... or a new kind of plugins.
-         3. update the Gateway and/or the management API assembly.xml files if necessary
+         2. add the plugin in the <dependencies> section with <type>zip</type> and <scope>runtime</scope>.
```

Comment:
👌🏻 it looks clear and understandable even for non Maven expert

Reply:
🙏🏻